easybuild/easyconfigs/t/TransDecoder/TransDecoder-2.1.0-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = "PackedBinary"
+
+name = "TransDecoder"
+version = '2.1.0'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://github.com/TransDecoder/TransDecoder/wiki'
+description = """TransDecoder identifies candidate coding regions within transcript sequences,
+ such as those generated by de novo RNA-Seq transcript assembly using Trinity,
+ or constructed based on RNA-Seq alignments to the genome using
+ Tophat and Cufflinks."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/TransDecoder/TransDecoder/archive/']
+sources = ['v%(version)s.tar.gz']
+
+checksums = ['681a8dd72e482cb322d74d8431ddcda61f25c53ad5453d0e22c02c92b2157b24']
+
+dependencies = [
+    ('Perl', '5.24.1'),
+    ('CD-HIT', '4.6.8'),
+]
+
+sanity_check_paths = {
+    'files': ['TransDecoder.%s' % x for x in ['Predict', 'LongOrfs']],
```

Comment:
no need to use a list comprehension here:

```python
    'files': ['TransDecoder.LongOrfs', 'TransDecoder.Predict'],

Reply:
@boegel Done